src/main/java/io/gravitee/alert/api/condition/StringCompareCondition.java

```diff
@@ -98,12 +98,12 @@ public static FilterBuilder matches(String property, String pattern, boolean ign
         return new FilterBuilder(property, Operator.MATCHES, pattern, ignoreCase);
     }
 
-    public static StringCondition.FilterBuilder startsWith(String property, String pattern) {
-        return startsWith(property, pattern, false);
+    public static FilterBuilder startsWith(String property, String property2) {
```

Comment:
Why do you rename `pattern` to `property2` ?
As we are looking at checking if the string is starting with something, it sounds better to me to keep pattern.
wdyt ?

Reply:
FP